bpf/lib/conntrack.h

```diff
@@ -230,16 +225,10 @@ static inline int __inline__ ct_lookup6(void *map, struct ipv6_ct_tuple *tuple,
 			if (skb_load_bytes(skb, l4_off + 12, &flags, 2) < 0)
 				return DROP_CT_INVALID_HDR;
 
-			if (unlikely(flags.syn && !flags.ack))
+			if (unlikely(flags.rst || flags.fin))
+				action = ACTION_CLOSE;
```

Comment:
Ah, nice. My only feedback was that I figure it shouldn't stick around as long as an established connection if it's closed. This will now be a 10s timeout from the time of FIN/RST. Should mean it's cleaned up within 20s. (GC just triggered, next GC in 10s will not clean up the entry, but the following will).

Reply:
Not required or possible in this patch but in theory hooking into sock_ops you could delete the entry on sock state. Obviously future work though.